### PR TITLE
Brainwashed brains now have smooth sprites

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Brainwashed brains are now truly smooth. Machine-washing brains (even without a skillchip) smoothes them out and adds an examine line.

image

## Why It's Good For The Game

Brainwashing is a funny feature and this is a way to add more flavor to it (and possibly make someone actually use it if they find the skillchip)

## Changelog
:cl:
image: Brainwashing victims now have smoothed out brain sprites without those pesky wrinkles
/:cl:

### CODE REVIEW

**Comment by TetrakoHateThis** [comment]
The 4am Brain level of smoothness.

**Comment by Pumpkinoe** [comment]
Your brain: Wrinkly :( Gross
My brain: Smooth :) No gross

**Comment by carlarctg** [comment]
add to felinids

**Comment by SmArtKar** [comment]
> add to felinids

they have small brains already

**Comment by norsvenska** [comment]
can you microwave the brain to dry it

File: code/modules/mob/living/brain/brain_item.dm [hunk 1]
```diff
@@ -373,6 +379,11 @@
 
 /obj/item/organ/internal/brain/machine_wash(obj/machinery/washing_machine/brainwasher)
 	. = ..()
+	if (can_smoothen_out && !smooth)
+		icon_state = initial(icon_state) + "-smooth"
```
Comment (ShizCalev) [inline code/modules/mob/living/brain/brain_item.dm:383]: icon changes need to be performed in update_icon(), not machine_wash()

File: code/modules/mob/living/brain/brain_item.dm [hunk 2]
```diff
@@ -40,6 +40,10 @@
 
 	/// Size modifier for the sprite
 	var/brain_size = 1
+	/// Can this brain become smooth after it gets washed
+	var/can_smoothen_out = TRUE
+	/// We got smooth
+	var/smooth = FALSE
```
Comment (ShizCalev) [inline code/modules/mob/living/brain/brain_item.dm:46]: mind renaming this to smooth_brain or something a little more verbose?

File: code/modules/mob/living/brain/brain_item.dm [hunk 3]
```diff
@@ -48,6 +52,8 @@
```
Comment (ShizCalev) [inline code/modules/mob/living/brain/brain_item.dm:52]:
```
/obj/item/organ/internal/brain/vv_edit_var(var_name, var_value)
	. = ..()
	if(var_name == NAMEOF(src, smooth))
		update_apperance()
```

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/brain/brain_item.dm [hunk 4]
```diff
@@ -373,6 +379,11 @@
 
 /obj/item/organ/internal/brain/machine_wash(obj/machinery/washing_machine/brainwasher)
 	. = ..()
+	if (can_smoothen_out && !smooth)
+		icon_state = initial(icon_state) + "-smooth"
+		smooth = TRUE
+		update_appearance()
+
```
Comment (ShizCalev) [inline code/modules/mob/living/brain/brain_item.dm:386]:
```suggestion
/obj/item/organ/internal/brain/machine_wash(obj/machinery/washing_machine/brainwasher)
	. = ..()
	if (can_smoothen_out)
		smooth = TRUE
		update_appearance()

/obj/item/organ/internal/brain/update_icon(updates)
	. = ..()
	icon_state = initial(icon_state) + "-smooth"

```

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**